labs/lab-2-javascript-game/index.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+
+<html>
+<head>
+	<link href="stylesheet.css" rel="stylesheet" type="text/css">
+	<script type="text/javascript" src="javascript.js"></script>
+	<link href="https://fonts.googleapis.com/css?family=VT323" rel="stylesheet">
+</head>
+
+<body>
+	<div id = "main">
+		<nav id="nav">
+			<label>Enter Player Name: <input type="text" id="first-input" /></label>
+			<label>Enter Enemy Name: <input type="text" id="second-input" /></label>
+			<button id="initialize">Battle!</button>
+		</nav>
+
+		<div id="main-game">
+			<div id="actions">
+				<button id="attack" class="attack">Attack?</button>
+				<button id="heal" class="heal">Heal?</button>
+				<button id="retreat" class="retreat">Retreat?</input>
+			</div>
+
+			<div id="results">
+				<div id="player-name"></div>
+				<div id="enemy-name"></div>
+			</div>
+
+			<div class="player">Player Stats:</div>
+
+			<div class="sliders">
+				<div class="output">Player Health<div id="player-health"></div></div>
+				<div class="output">Player Heal Count<div id="heal-count"></div></div>
+				<div class="output">Player Wins<div id="player-wins"></div></div>
+			</div>
+
+			<div class="player">Enemy Stats:</div>
+
+			<div class="sliders">
+				<div class="output">Enemy Health<div id="enemy-health"></div></div>
+			</div>
+		</div>
+	</div>
+
+	<div id="end-game">
+		<p> Player has retreated! </p>
```

Comment:
(You could sneak the player's name and the enemy's name into these end game screens, too)